docs/4.0/components/modal.md

```diff
@@ -508,7 +508,7 @@ Modals have two optional sizes, available via modifier classes to be placed on a
     <div class="modal-content">
 
       <div class="modal-header">
-        <h4 class="modal-title" id="myLargeModalLabel">Large modal</h4>
+        <h5 class="modal-title h4" id="myLargeModalLabel">Large modal</h5>
```

Comment:
@mdo: not sure if you agree with this solution either...

Reply:
These changes are fine, but I'm unsure why you're adding them to this PR. What problem is this solving that's tied to https or broken links?

Reply:
Because they are added to the TOC, which in turn results in non-existent fragments...

Just try it for yourself locally:

```
http://127.0.0.1:9001/docs/4.0/components/modal/#large-modal
http://127.0.0.1:9001/docs/4.0/components/modal/#small-modal
```

Or try clicking on the generated TOC items and see they don't go anywhere ;)